fw/tls.c

```diff
@@ -186,12 +178,15 @@ tfw_tls_msg_process(void *conn, TfwFsmData *data)
 		 * Pass tls->io_in.skb_list to data_up ownership for the upper
 		 * layer processing.
 		 */
-		r = tfw_tls_chop_skb_rec(tls, tls->io_in.skb_list, &data_up);
+		r = tfw_tls_chop_skb_rec(tls);
```

Comment:
Do we need the function at all? It seems we can call `ss_skb_list_chop_head_tail()` just here

Reply:
@krizhanovsky Dropped